org.eclipse.wildwebdeveloper.xml/src/org/eclipse/wildwebdeveloper/xml/internal/ui/messages.properties

```diff
@@ -21,6 +21,9 @@ XMLCatalogPreferencePage_OpenInEditorMessage=Do you want to Apply the changes an
 XMLCatalogPreferencePage_OpenInEditorApplyAndEdit=Apply and Edit
 XMLCatalogPreferencePage_OpenInEditorNo=No
 
+# XML CodeLens preference page
+XMLCodelensPreferencePage_codelens_enabled=Enable/disable XML &CodeLens
```

Comment:
Just "Enable XML Codelens" as it's a checkbox, ideally also with a description of what are the expected codelens (reference count on schemas)

Reply:
> Just "Enable XML Codelens" as it's a checkbox, ideally also with a description of what are the expected codelens (reference count on schemas)

LemMinx provides this feature but LemMinx extension could provide Codelens too (ex : Codelens to execute a goal for maven), so it's not only restricted for XSD and DTD.

Reply:
OK. It's more the "Enable/Disable" part which is not good UI workding besides a checkbox.